src/app/util/Braintree/Braintree.js

```diff
@@ -24,14 +26,25 @@ export class Braintree {
     async create() {
         const { default: dropIn } = await import('braintree-web-drop-in');
         const authorization = await this.requestBraintreeClientToken();
+        const configuration = await this.requestBraintreeConfig();
+        this.isThreeDSecure = configuration.is_three_d_secure;
         this.braintreeDropIn = await dropIn.create({
             authorization,
-            container: `#${ this.containerId }`
+            container: `#${ this.containerId }`,
+            threeDSecure: this.isThreeDSecure
         });
 
         return true;
     }
 
+    requestBraintreeConfig() {
+        const query = new Field('getBraintreeConfig').addFieldList(['is_three_d_secure']);
```

Comment:
Prefer always having the queries separately from other stuff, in a separate .query.js file.
I will allow this here because that's the way it'd already been written before you in the same functionality. Nevertheless, remember this for further work.